pkgs/applications/graphics/sane/backends/default.nix

```diff
@@ -1,9 +1,9 @@
 { callPackage, fetchurl, ... } @ args:
 
 callPackage ./generic.nix (args // rec {
-  version = "1.0.25";
+  version = "1.0.27";
   src = fetchurl {
-    sha256 = "0b3fvhrxl4l82bf3v0j47ypjv6a0k5lqbgknrq1agpmjca6vmmx4";
+    sha256 = "1j9nbqspaj0rlgalafb5z6r606k0i22kz0rcpd744p176yzlfdr9";
```

Comment:
When I tried it on my system, I actually had to revert the hash back to its previous version.

So oddly the following combination worked for me:

```
version = "1.0.27";
   src = fetchurl {
   sha256 = "0b3fvhrxl4l82bf3v0j47ypjv6a0k5lqbgknrq1agpmjca6vmmx4";
```